software/runtime/runtime.mk

```diff
@@ -63,6 +63,8 @@ DEFINES += -DPRINTF_DISABLE_SUPPORT_FLOAT -DPRINTF_DISABLE_SUPPORT_LONG_LONG -DP
 DEFINES += -DNUM_CORES=$(num_cores)
 DEFINES += -DNUM_GROUPS=$(num_groups)
 DEFINES += -DNUM_CORES_PER_TILE=$(num_cores_per_tile)
+DEFINES += -DNUM_CORES_PER_GROUP=$(shell awk 'BEGIN{print $(num_cores)/$(num_groups)}')
+DEFINES += -DNUM_TILES_PER_GROUP=$(shell awk 'BEGIN{print $(num_cores_per_group)/$(num_cores_per_tile)}')
```

Comment:
The parameters seem redundant.

Reply:
Solved by commit 9c04ba81533980b64b11edd43871aa480637400d ?